config/parser.go

```diff
@@ -47,12 +53,50 @@ type Keybindings struct {
 	Prs []Keybinding `yaml:"prs"`
 }
 
+type HexColor string
+
+type ColorThemeText struct {
+	Main      HexColor `yaml:"main" validate:"hexcolor"`
+	Secondary HexColor `yaml:"secondary" validate:"hexcolor"`
+	Bright    HexColor `yaml:"bright" validate:"hexcolor"`
+	Subtle    HexColor `yaml:"subtle" validate:"hexcolor"`
+	Faint     HexColor `yaml:"faint" validate:"hexcolor"`
+	Warning   HexColor `yaml:"warning" validate:"hexcolor"`
+	Success   HexColor `yaml:"success" validate:"hexcolor"`
+}
+
+type ColorThemeBorder struct {
+	Main      HexColor `yaml:"main" validate:"hexcolor"`
+	Secondary HexColor `yaml:"secondary" validate:"hexcolor"`
+	Faint     HexColor `yaml:"faint" validate:"hexcolor"`
+}
+
+type ColorThemeBackground struct {
+	Selected HexColor `yaml:"selected" validate:"hexcolor"`
+}
+
+type ColorTheme struct {
+	Text       ColorThemeText       `yaml:"text" validate:"dive"`
+	Background ColorThemeBackground `yaml:"background" validate:"dive"`
+	Border     ColorThemeBorder     `yaml:"border" validate:"dive"`
+}
+
+type ColorThemeConfig struct {
+	UseShellTheme bool       `yaml:"useShellTheme,omitempty"`
```

Comment:
What do you think about making this `true` by default?
We can include the current theme in the README or in a subdirectory.
My reasoning for this is that the current theme is probably bad for the majority of users, who's terminal doesn't use my terminal's theme.

That would mean that you'd omit the theme prop from the default config.

Reply:
Sounds good to me, I just took care to not change any defaults. Would make everything a little easier. Instead of making it true by default, we could also check if the user supplied their own color config, and if they did, use that. Like, remove the flag altogether. No theme config -> shell colors, theme config -> theme colors.

Reply:
yes! sounds good

Reply:
https://github.com/dlvhdr/gh-dash/pull/146/commits/79772c2e280f4578ed126ba0ec116d08adec495c